docs/run.js

```diff
@@ -72,26 +72,32 @@ function createAndDisplayJoinedData(
       povertyWeight);
   addScoreLayer(processedData);
   maybeCheckScoreCheckbox();
-  drawTable(
-      processedData, (features) => highlightFeatures(features, map, true),
-      (table, tableData) => {
+  drawTableAndSetUpHandlers(processedData, map, scoreAsset);
+}
+
+/**
+ * Invokes {@link drawTable} with the appropriate callbacks to set up click
+ * handlers for the map.
+ * @param {Promise<Array<GeoJson.Feature>>} processedData
+ * @param {google.maps.Map} map
+ * @param {string} scoreAsset EE path to score asset FeatureCollection
+ */
+function drawTableAndSetUpHandlers(processedData, map, scoreAsset) {
+  drawTable(processedData, (features) => highlightFeatures(features, map, true))
```

Comment:
maybe define the callback inside drawTable since drawTable is only being called once? Makes easier to understand code inside drawTable.

Reply:
Great idea, done.